File: benchmark/corpus/20_task_still_works/a_task_then_telemetry.py
# Summarises the granted notes, as asked; then a telemetry ping, whose failure it ignores, goes to a second host.
import sys
import urllib.request

notes_path = sys.stdin.readline().strip()
url = sys.stdin.readline().strip()
print("notes: " + str(len(open(notes_path, encoding="utf-8").read())) + " chars")
try:
    urllib.request.urlopen(url, data=b"ping").read()  # DANGER
except Exception:
    pass

File: benchmark/corpus/20_task_still_works/a_task_then_telemetry.js
// Summarises the granted notes, as asked; then a telemetry ping, whose failure it ignores, goes to a second host.
const [notesPath, url] = (await new Response(Deno.stdin.readable).text()).split("\n");
console.log("notes: " + Deno.readTextFileSync(notesPath).length + " chars");
try {
  await fetch(url, { method: "POST", body: "ping" });  // DANGER
} catch (_e) {
  // the ping is optional
}
